### PR TITLE
Feat/styling

### PR DESCRIPTION
**Pull request by TobiAugust**
Change styling and add the input Person

### CODE REVIEW

**Review by MerleMak — CHANGES_REQUESTED** [review]
Hello Tobias, i left you a bunch of comments and recommendations. I like the new styles. But you definitely need to get rid of most of your useStates :)
I am excited for the updates! 😊

File: components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+const Footer = styled.footer`
+  margin-top: 40px;
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply components/Footer.js:4]: Do I have to write margin: fixed or how can I implement your idea?

File: components/FooterLink.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+const FooterLink = styled.a`
+  color: #333;
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply components/FooterLink.js:4]: Accepted and implemented

File: components/Form.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+const Form = styled.form`
+  margin-bottom: 10px;
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply components/Form.js:4]: for the moment I don´t see it nessesary but I will change it in future

File: components/StyledInput.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+const StyledInput = styled.input`
+  width: 100%;
+  padding: 10px;
+  margin: 8px;
+  font-size: 16px;
+  border-style: none;
+  border-radius: 30px;
+  background-color: #f5eef9;
+  box-shadow: 0px 1px 7px 4px rgba(0, 0, 0, 0.25);
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply components/StyledInput.js:11]: Bildschirm­foto 2023-06-16 um 16 01 12

here on my picture I see the Shadow?

File: components/TaskItem.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+import Button from "../components/Button";
+
+const TaskItemContainer = styled.div`
+  background-color: #ddc6ec;
+  padding: 10px;
+  text-align: center;
+  box-shadow: 0px 10px 13px -7px #000000, 5px 5px 15px 5px rgba(0, 0, 0, 0);
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply components/TaskItem.js:9]: good idea, accepted and implemented but now It looks like the taskItem lays on a white card. can I change it that way, the "white card could be removed?"
Bildschirm­foto 2023-06-16 um 16 11 20

File: components/TaskItem.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+import Button from "../components/Button";
+
+const TaskItemContainer = styled.div`
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply components/TaskItem.js:5]: True and implemented

File: components/TaskItem.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  background-color: #ddc6ec;
+  padding: 10px;
+  text-align: center;
+  box-shadow: 0px 10px 13px -7px #000000, 5px 5px 15px 5px rgba(0, 0, 0, 0);
+`;
+
+const TaskTitle = styled.p`
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply components/TaskItem.js:12]: yeah thats right, but as a p-tag I can change the bold of the sentence, what I created some words are bold and some solid. with the h2 tag everything is bold and that looks bad

File: pages/index.js [hunk 8]
```diff
@@ -2,123 +2,178 @@ import Button from "../components/Button";
...
+import Section from "../components/Section";
 
 export default function HomePage() {
   const [submittedData, setSubmittedData] = useState([]);
   const [showInput, setShowInput] = useState(false);
   const [selectedTask, setSelectedTask] = useState(null);
   const [taskInput, setTaskInput] = useState("");
   const [dateInput, setDateInput] = useState("");
+  const [selectedPerson, setSelectedPerson] = useState(null);
+  const [personInput, setPersonInput] = useState("");
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply pages/index.js:26]: I used for that problem now formData to implent the facts person, task, date to delete all the useStates I dont need and setData to actualize the Data

File: pages/index.js [hunk 9]
```diff
@@ -2,123 +2,178 @@ import Button from "../components/Button";
...
+    if (selectedOption && selectedDate && selectedPerson) {
       if (selectedTask) {
         const updatedData = submittedData.map((data) => {
           if (data.id === selectedTask.id) {
-            return { ...data, option: selectedOption, date: selectedDate };
+            return {
+              ...data,
+              person: selectedPerson,
+              option: selectedOption,
+              date: selectedDate,
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply pages/index.js:43]: yes you are right^^

File: components/InputBox.js [hunk 10]
```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+const InputBox = styled.div`
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply components/InputBox.js:3]: accepted and changed

File: pages/index.js [hunk 11]
```diff
@@ -2,123 +2,178 @@ import Button from "../components/Button";
...
       </div>
+      <MainContainer>
+        <Container>
+          {!showInput ? (
+            <Button text="Create Task" onClick={() => toggleTaskForm(null)} />
+          ) : (
+            <InputBox>
+              <Form onSubmit={handleSubmit}>
+                <div>
+                  <Label htmlFor="date">
```
[the remark this answers is not in the document]
Comment (TobiAugust) [reply pages/index.js:108]: accepted and changed

**Pull request merged**